content/resources/csc207.md

```diff
@@ -7,30 +7,47 @@ tags:
   - utm-cs
 ---
 
-# CSC207: Intro to Software Design
+# CSC207: Software Design
+#### Prerequisite: 60% in [CSC148](./csc148)
 
-## Object Oriented Programming
+<grid-1-x-2 title="Fall 2020 Class Website" img-src="https://i.imgur.com/jYTYgeu.png" link="https://axiom.utm.utoronto.ca/~207/20f/" desc="All credits to Sadia Sharmin and Larry Zhang" button="Check it out!"></grid-1-x-2>
 
-### A gentle intro to Object Oriented Programming
+### What is CSC207?
+CSC207 is an introduction course where students will learn popular design patterns, as well as developments tools and methods. such as version control and the basic of agile methodology. Students will also learn how to use a new programming language, specifically Java and the popular IDE Eclipse.
```

Comment:
Perhaps we can change the last sentence to, "Students will learn how to program in Java using the Eclipse IDE"